### ISSUE TITLE
Day Interval feature IOS < 10

### ISSUE THREAD

**Issue by RobertBrunhage** [MaikuB/flutter_local_notifications#54]
Sorry to disturb your again, If I would ignore IOS version < 10. How would I implement a day interval for notifications?

Ex: every other day, every third day, every second week?

**Comment by MaikuB** [comment]
As mentioned in the other thread this is not a feature the plugin is supporting and you'd need to modify a copy of the plugin code to do what to do what you're after. The issues board here is really raising for bugs found in features of the plugin or feature requests. 

Presumably, you're looking to make modifications now and as such should posting your questions on other places like stack overflow, debugging the plugin code to get ideas, consulting the iOS documentation and possibly the Flutter.

In summary, sorry but it's not something I'll be providing support for as if maintainers such as myself were to provide support on other people's forks then that would just lead to burnout

**Comment by MaikuB** [comment]
Forgot to add, this is the iOS documentation that you should check to get you started https://developer.apple.com/documentation/usernotifications
